pkg/storage/simple/manager.go

```diff
@@ -74,15 +75,26 @@ func (factory *simpleSpaceManagerFactory) Create(parameters map[string]interface
 		return nil, ErrorInternalUnknown.Format(err)
 	}
 	// create storage driver
-	storageDriverName, ok := parameters["storagedriver"]
+	storageDriverName, ok := parameters[common.ParameterNameStorageDriver]
 	if !ok {
-		return nil, ErrorContentMissing.Format("storagedriver")
+		return nil, ErrorContentMissing.Format(common.ParameterNameStorageDriver)
 	}
 	storageDriver, err := driver.Create(fmt.Sprint(storageDriverName), parameters)
 	if err != nil {
 		return nil, ErrorInternalUnknown.Format(err)
 	}
-	return NewSpaceManager(storageDriver, locker, lock.TimeoutImmediate), nil
+
+	var lockTimeout = lock.TimeoutInfinite
```

Comment:
`lockTimeout` defaults to `lock.TimeoutImmediate`.  If a user want a longer timeout, he can set `locktimeout` in config.